### ISSUE TITLE
Fix SyntaxError when an empty string is used for a rule's custom message 

### ISSUE THREAD

**Issue by mems** [stylelint/stylelint#3718]
If you declare empty custom message for a rule, stylelint fail with an error

> Which rule, if any, is the bug related to?

Any

> What CSS is needed to reproduce the bug?

Any

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "property-no-vendor-prefix": [true, {"message": ""}],
  }
}
```

> Which version of stylelint are you using?

9.6.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Node API (via [gulp-stylelint](https://www.npmjs.com/package/gulp-stylelint))

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

No SyntaxError exception, at least an error about misconfiguration (eg. "Custom message shouldn't be empty")

> What actually happened (e.g. what warnings or errors did you get)?

```shell
SyntaxError: Invalid regular expression: /(^.{1,1}(\s+|$))|(^.{1,0}(\\|/|_|\.|,|;|-))/: numbers out of order in {} quantifier
    at new RegExp (<anonymous>)
    at wrapWord (<redacted>\node_modules\table\dist\wrapWord.js:24:14)
    at calculateCellHeight (<redacted>\node_modules\table\dist\calculateCellHeight.js:36:34)
    at cells.forEach (<redacted>\node_modules\table\dist\calculateRowHeightIndex.js:35:66)
    at Array.forEach (<anonymous>)
    at rows.forEach.cells (<redacted>\node_modules\table\dist\calculateRowHeightIndex.js:26:11)
    at Array.forEach (<anonymous>)
    at calculateRowHeightIndex (<redacted>\node_modules\table\dist\calculateRowHeightIndex.js:24:8)
    at Object.table (<redacted>\node_modules\table\dist\table.js:99:63)
    at formatter (<redacted>\node_modules\stylelint\lib\formatters\stringFormatter.js:135:6)
    at results.reduce (<redacted>\node_modules\stylelint\lib\formatters\stringFormatter.js:176:15)
    at Array.reduce (<anonymous>)
    at module.exports (<redacted>\node_modules\stylelint\lib\formatters\stringFormatter.js:163:20)
    at module.exports (<redacted>\node_modules\stylelint\lib\formatters\verboseFormatter.js:8:16)
    at reporter <redacted>\node_modules\gulp-stylelint\dist\reporter-factory.js:58:25)
    at <redacted>\node_modules\gulp-stylelint\dist\index.js:151:14
```

**Comment by jeddy3** [comment]
@mems Thanks for the clear report and for using the template.

This definitely looks like a bug (likely in the [string formatter](https://github.com/stylelint/stylelint/blob/master/lib/formatters/stringFormatter.js)).  As the issue has been labelled "help wanted", please consider [contributing a fix](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#fixing-a-bug-in-an-existing-rule) if you'd like to help out.
